.github/linters/linters_config.toml

```diff
@@ -0,0 +1,8 @@
+[tool.black]
```

Comment:
Does linters_config.toml have semantics of pyproject.toml? https://www.python.org/dev/peps/pep-0518/ If yes, maybe it is worth to put it in repo root (and name it pyproject.toml) so that running linters locally is easier.

Reply:
See: https://github.com/github/super-linter/blob/master/docs/using-rules-files.md -> LINTER_RULES_PATH

Reply:
Yes it have semantics of pyproject.toml. I can move it to repo root.